### PR TITLE
[ModelZoo] Test model works with sqlflow and edl

### PR DESCRIPTION
**Pull request by typhoonzero**
Test run ElasticDL with current models. 
Depends on https://github.com/sql-machine-learning/models/pull/26.

### CODE REVIEW

**Review by terrytangyuan — COMMENTED** [review]
ElasticDL can work with ODPS tables directly so `scripts/convert_iris_to_recordio.py` is not necessary.  Here's an example SQL query that would work with iris dataset: https://github.com/sql-machine-learning/sqlflow/blob/develop/cmd/sqlflowserver/main_test.go#L768

Or you can run it through ElasticDL CLI directly: https://github.com/sql-machine-learning/elasticdl/blob/develop/scripts/client_test.sh#L67-L87

**Comment by typhoonzero** [comment]
> ElasticDL can work with ODPS tables directly so `scripts/convert_iris_to_recordio.py` is not necessary. Here's an example SQL query that would work with iris dataset: https://github.com/sql-machine-learning/sqlflow/blob/develop/cmd/sqlflowserver/main_test.go#L768
> 
> Or you can run it through ElasticDL CLI directly: https://github.com/sql-machine-learning/elasticdl/blob/develop/scripts/client_test.sh#L67-L87

Thanks for reminding, will try that out.

File: .travis.yml [hunk 1]
```diff
@@ -4,17 +4,32 @@ branches:
...
 install:
 - python setup.py -q install
-
 script:
 - python setup.py -q test
+
+jobs:
+  include:
+  - stage: ElasticDLTest
+    script:
```
Comment (tonyyang-svail) [inline .travis.yml:22]: Do we need `set -e`?

**Review by tonyyang-svail — APPROVED** [review]
Generally LGTM.

**Review by terrytangyuan — APPROVED** [review]

**Pull request merged**